Extension/src/telemetry.ts

```diff
@@ -79,30 +78,12 @@ export function getExperimentationService(): Promise<IExperimentationService> |
     return initializationPromise;
 }
 
-export async function showLanguageStatusExperiment(): Promise<boolean> {
-    return isExperimentEnabled("ShowLangStatBar");
-}
-
-export async function showStatusBarIntelliSenseButton(): Promise<boolean> {
-    const result: boolean = await isExperimentEnabled("showStatusBarIntelliSenseIndicator");
-    return result;
-}
-
-async function isExperimentEnabled(experimentName: string): Promise<boolean> {
```

Comment:
Don't delete this function. We'll use it again the next time we have an experiment.